client/src/helpers/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const instance = axios.create({
+    baseURL: 'http://localhost:3000/',
+})
+
+instance.interceptors.request.use(
```

Comment:
this is great, I love this easy setup you made here. Great job setting this up, would totally make life much easier for your team in the long run during a project I think.